src/main/java/com/shapesecurity/shift/parser/Parser.java

```diff
@@ -81,11 +85,27 @@ private void consumeSemicolon() throws JsError {
 
     @NotNull
     protected <T extends Node> T markLocation(@NotNull SourceLocation startLocation, @NotNull T node) {
-        // TODO: actually mark location
-//    node.loc = Maybe.just(new SourceSpan(Maybe.nothing(), startLocation, new SourceLocation(this.lastLine+1, this.lastIndex-this.lastLineStart, this.lastIndex)));
+        locations.put(
+                node,
```

Comment:
No, I used spaces. While I was debugging I tried splitting up the parameters so I could see better what the problem might be. If you'd like I'm happy to revert it into one single line. I did not modify anything here besides whitespace.